streaming-connect-sink/src/main/java/com/adobe/platform/streaming/sink/impl/AEPPublisher.java

```diff
@@ -87,7 +94,27 @@ public void publishData(List<Pair<String, SinkRecord>> messages) throws AEPStrea
       );
 
       count++;
-      LOG.debug("Successfully published data to Adobe Experience Platform: {}", response);
+
+      if (!response.isEmpty() && response.hasNonNull(RESPONSE_CODE_KEY) &&
+        response.get(RESPONSE_CODE_KEY).asInt() == HttpStatus.SC_MULTI_STATUS) {
+        JsonNode publishMessagesResponses = response.get(RESPONSES_KEY);
+        for (JsonNode messageResponse : publishMessagesResponses) {
+          if (messageResponse.hasNonNull(STATUS_KEY)) {
+            failedMessageCount++;
+            LOG.debug("Failed to publish message to Adobe Experience Platform: {}", messageResponse);
+          } else {
+            successMessageCount++;
+          }
+        }
+        totalMessageCount = successMessageCount + failedMessageCount;
+
+        LOG.debug("Total published message count = {}. Success message count = {}. Failed " +
+          "message count = {}.", totalMessageCount, successMessageCount, failedMessageCount);
+
+      } else {
+        LOG.debug("Response of the data sent to Adobe Experience Platform: {}", response);
```

Comment:
Is there a case where responseCode is not 207 ?
Should it be marked as error in sending data to inlet.

Reply:
No responseCode other than 207 is supported as of now . We have already handled all the possible error code in httpConnection . Should we ignore this message ?